vp-docs/guide/advanced/grouped-table.md

```diff
@@ -151,3 +151,20 @@ In this case header row expects a value for each column
 - The column object can be accessed via `props.column`
 - You can access the formatted row data (for example - formatted date) via `props.formattedRow`
   :::
+
+## Collapsable Rows
+
+To allow the row to collapse and expand you can use the groupOption "collapsable". You can either pass in a boolean or a number. 
+If "collapsable" is set to "true" then it will default to making the first column collapsable. Otherwise, you can specify the column index number.
```

Comment:
```suggestion
If `collapsable` is set to `true` then it will default to making the first column collapsable. Alternatively, you can specify the column index number.
```